dev-docs/bidders/viously.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bidder
+title: Viously
+description: Prebid Viously Bidder Adaptor
+pbjs: true
+pbs: true
```

Comment:
I don't see a code PR supporting Prebid Server. Please set this to false. Also, you're missing some other metadata fields like multiformat_supported, ortb_blocking_supported, dchain, etc. Please check https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter